src/Apps/Sell/Routes/PurchaseHistoryRoute.tsx

```diff
@@ -11,15 +20,30 @@ import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
 const FRAGMENT = graphql`
   fragment PurchaseHistoryRoute_submission on ConsignmentSubmission {
     provenance
+    signature
   }
 `
 
 const Schema = Yup.object().shape({
   provenance: Yup.string().trim(),
+  signature: Yup.boolean(),
 })
 
+export const PROVENANCE_LIST = [
```

Comment:
I wonder if we could move it to a separate helper file to keep the component file a bit smaller. Something like `src/Apps/Sell/Utils/formUtils.ts`.